### PR TITLE
CIS update blog post

### PR DESCRIPTION
**Pull request by sej7278**
v3.0.0 benchmark was released for AlmaLinux 8 @bennyvasquez

### CODE REVIEW

File: content/blog/2023-10-31-cis-benchmarks-update.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: "CIS Benchmarks Update"
+type: blog
+author:
+ name: "Simon John"
+ bio: "Security Standards Architect"
+ image: /users/sjohn.png
+date: '2023-10-31'
```
Comment (bennyvasquez) [inline content/blog/2023-10-31-cis-benchmarks-update.md:8]:
```suggestion
date: '2023-11-20'
images:
  - /blog-images/cis_update.svg
```

File: content/blog/2023-10-31-cis-benchmarks-update.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Many organisations use the CIS benchmarks as their baseline for security hardening operating systems and applications. They can be either a starting point to build on or a goal to aspire towards - traditionally you'd have a percentage pass rate to achieve based on your organisation's policies for example "at least 70% at Level 2". Being an industry standard means that you can find plenty of tutorials, tools and experienced practitioners to help you.
+
+The benchmarks cover topics like which SSH ciphers to use, AAA best practices like password complexity and filesystem permissions, how to best configure a host-based firewall (like firewalld/iptables) and how to setup auditd and rsyslog to ensure that you're logging enough information and have an audit trail in case of a security incident, and how to stop disabling SELinux! The DISA [STIGs](https://public.cyber.mil/stigs/) are very similar, but with some specific additions for national security and a different delivery method.
+
+The benchmarks are just documents, you can read them as PDF's, no special software to install. Or you can browse them on the website. They're available to all for free with no paywall.
```
Comment (bennyvasquez) [inline content/blog/2023-10-31-cis-benchmarks-update.md:25]:
```suggestion
The benchmarks are just documents - you can read them as PDF's - no special software to install. You can find them on the website. They're available to all for free with no paywall.
```

Comment (bennyvasquez) [reply content/blog/2023-10-31-cis-benchmarks-update.md:25]: If we can link to them directly here, we should.

File: content/blog/2023-10-31-cis-benchmarks-update.md (hunk 1 above)
Comment (bennyvasquez) [inline content/blog/2023-10-31-cis-benchmarks-update.md:8]:
```suggestion
date: '2023-11-28'
```

Comment (bennyvasquez) [reply content/blog/2023-10-31-cis-benchmarks-update.md:8]: I realized next week will be 8.9 and then thanksgiving in the US, so we'll want to do this one the week after.

File: content/blog/2023-10-31-cis-benchmarks-update.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+[Tenable](https://www.cisecurity.org/partner/tenable)'s Nessus Pro includes compliance audits against published standards and baselines (CIS, DISA STIGs) and across more than 15+ security frameworks (HIPAA, NIST 800-53). The dedicated compliance team comes with decades of combined experience and are an active part of industry working groups such as CIS. I recently worked with Tenable Research to improve the accuracy of the AlmaLinux OS 9 CIS audits.
+
+[Qualys](https://www.cisecurity.org/partner/qualys) Policy Compliance assesses various CIS benchmarks - including AlmaLinux, as well as prioritises the risk of the misconfiguration by mapping them to the MITRE ATT&CK framework and threats such as ransomware, with the ability to remediate misconfigurations to reduce risk and bring servers into compliance.
+
+## How can I security harden my servers?
+
+On my [github](https://github.com/sej7278/virt-installs) I have a few scripts for hardening various distro's including AlmaLinux, to CIS benchmarks, using some of my favourite technologies - Kickstart and libvirt, but also Ansible, cloud-init and Preseed.
```
Comment (bennyvasquez) [inline content/blog/2023-10-31-cis-benchmarks-update.md:48]:
```suggestion
On my [GitHub](https://github.com/sej7278/virt-installs) I have a few scripts for hardening various distro's including AlmaLinux, to CIS benchmarks, using some of my favourite technologies - Kickstart and libvirt, but also Ansible, cloud-init and Preseed.
```

File: content/blog/2023-10-31-cis-benchmarks-update.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+With CIS [SecureSuite](https://www.cisecurity.org/cis-hardened-images) membership you can get access to pre-hardended virtual machine images for most cloud providers, build-kits for hardening existing infrastructure and the CIS-CAT Pro tool for testing.
+
+Unlike some security frameworks which can be quite woolly and high-level, the CIS benchmarks are distribution-specific and very descriptive - they tell you exactly which commands to type or what to put in a config file to remediate non-compliances and how to retest. This is what most companies I've worked with find attractive - a sysadmin can do the hardening and a SOC analyst can measure compliance without hiring a consultancy to tell you what is meant by phrases like *"use sufficient cryptographic controls for the classification of data at rest and in transit"*.
```
Comment (bennyvasquez) [inline content/blog/2023-10-31-cis-benchmarks-update.md:52]: I've not heard the word woolly used in this context before and am not sure what it means for you. If it's too colloquial we should adjust it to a more common word, or maybe just add more context.

File: content/blog/2023-10-31-cis-benchmarks-update.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+The AlmaLinux OS 9 v1.0.0 benchmark was published in December 2022, which was actually the upstream of the RHEL 9 v1.0.0 benchmark that was published a couple of weeks *earlier!* We're currently working on the F34 family vNext benchmark which will become the next AlmaLinux OS 9 benchmark update.
+
+It's truly a collaborative community effort, with representatives from various distro's as well as end-users and security professionals from all over the world, regularly attending the [Weekly Webex](https://workbench.cisecurity.org/communities/1) with the folks at CIS. I attend the distribution-independent Linux calls, where we also work on Debian, Ubuntu and even recently Amazon Linux. It can be a great learning experience too.
+
+You can also help by reviewing [vNext drafts](https://workbench.cisecurity.org/benchmarks/10093) and adding a comment/ticket or even starting a discussion - maybe you don't agree with a policy or could improve upon it or have even just found a typo. It's not all Linux, there's plenty of other [communities](https://workbench.cisecurity.org/communities/public) you may be interested in too, including IoT, Apache, macOS, Cisco....
```
Comment (bennyvasquez) [inline content/blog/2023-10-31-cis-benchmarks-update.md:76]:
```suggestion
You can also help by reviewing [vNext drafts](https://workbench.cisecurity.org/benchmarks/10093) and adding a comment/ticket or even starting a discussion - maybe you don't agree with a policy or could improve upon it or have even just found a typo. It's not all Linux, there's plenty of other [communities](https://workbench.cisecurity.org/communities/public) you may be interested in too, including IoT, Apache, macOS, Cisco....

If you just want to come chat about all things AlmaLinux (and a surprising number of pet-related conversations), join us at [chat.almalinux.org](https://chat.almalinux.org)
```

**Review by bennyvasquez — COMMENTED** [review]
This is a GREAT post, @sej7278! I'm so excited to get this out, and thank you for your patience.

File: content/blog/2023-10-31-cis-benchmarks-update.md (hunk 4 above)
Comment (sej7278) [reply content/blog/2023-10-31-cis-benchmarks-update.md:52]: Changed to "vague"

File: content/blog/2023-10-31-cis-benchmarks-update.md (hunk 1 above)
Comment (sej7278) [reply content/blog/2023-10-31-cis-benchmarks-update.md]: you might want to rename the file eventually too

**Comment by sej7278** [comment]
i can squash this and/or do a new PR if you want, i realised too late i was committing to my master so had to re-sync

**Comment by sej7278** [comment]
moved to #385

**Pull request closed without merging**